Test.C

```diff
@@ -1,53 +1,161 @@
+#include <stdio.h>
+#include <stdlib.h>
+
 /*
 
-Hello
+defining number literals
 
 */
 
-int main(){ 
-
-    //initialize game 
+#define NAME_LENGTH 20
+#define DESCRIPTION_LENGTH 200
+#define NUM_EQUIPMENT 10
+#define NUM_ENCRYPTION 10
+#define NUM_ATTACK 5
+
+int main(){
+
+    //initialize game
+
+    struct Equipment {
+        int id;
+        char name[NAME_LENGTH]; // name of the piece of Equipment
+        char description[DESCRIPTION_LENGTH]; // short description of Equipment
+        int credits; // cost of the Equipment
+        int effectiveness; // number of messages able to be sent
+    };
+
+    struct Encryption {
+        int id;
+        char name[NAME_LENGTH]; // name of the encryption method
+        char description[DESCRIPTION_LENGTH]; // short description of how encryption works
+        int credits; // cost of the encryption
+        double effectiveness; // how effective the encryption is (percentage)
+
+    };
+
+    struct Attack {
+        int id;
+        char name[NAME_LENGTH]; // name of the attack
+        char description[DESCRIPTION_LENGTH]; // short description of how the attack works
+        int tier; // level of intensity
+
+    };
+
+    struct EquipmentInventory {
+        Equipment equipment; // the data
+        int quantity; // the number of this equipment type currently owned
+        EquipmentInventory *next; // the pointer
+    };
+
+    struct EncryptionInventory {
+        Encryption encryption; // the data
+        EncryptionInventory *next; // the pointer
+    };
+
+    struct Player {
+        char name[NAME_LENGTH]; // the player's name
+        EquipmentInventory equipmentInventory; // the current equipment the user owns
```

Comment:
Replace with `int equipmentInventory[9];'
```suggestion
        int equipmentInventory[NUM_EQUIPMENT] = {0}; // the current equipment the user owns
```